app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from typing import Optional
+
+
+def copy_file(command: str) -> Optional[str]:
+
+    original_file = command.split(" ")[1]
+    copied_file = command.split(" ")[2]
+
+    if original_file == copied_file:
+        return None
```

Comment:
Also if you call just `return` is will return `None` too : ) So that None a bit redundat imho